packages/cadl-vscode/test/utils.ts

```diff
@@ -18,7 +18,7 @@ async function createOnigLib(): Promise<IOnigLib> {
 const registry = new Registry({
   onigLib: createOnigLib(),
   loadGrammar: async (scopeName) => {
-    const data = await readFile("./dist/cadl.tmLanguage");
```

Comment:
Fun stuff with filesystem taking cwd into account :)

Reply:
For some definition of "fun" 🤣